plonetheme/onegovbear/theme/scss/elements/form.scss

```diff
@@ -0,0 +1,9 @@
+input[type="text"],
+input[type="password"],
+textarea {
```

Comment:
The `textarea` does not need to include `input`, since it includes `textarea` below which also includes `input`.
-> https://github.com/4teamwork/ftw.theming/pull/25/files#diff-627e2046dc100b9a8a282d53df2d9941R37